### DISCUSSION TITLE
v4.1.0

### DISCUSSION THREAD

**Discussion by debugtalk** [httprunner/httprunner#1321] [0 - 公告（Announcements）]
- feat: add `wiki` sub-command to open httprunner website
- feat: add `build` sub-command for function plugin

**go version**

- feat #1268: convert postman collection to HttpRunner testcase
- feat #1291: run testcases in v2/v3 JSON/YAML format with hrp run/boom command
- feat #1280: support creating empty scaffold project
- fix #1308: load `.env` file as environment variables
- fix #1309: locate plugin file upward recursively until system root dir
- fix #1315: failed to generate a report in failfast mode
- refactor: move base_url to config `environs`
- refactor: implement testcase conversions with `hrp convert`

**Full Changelog**: https://github.com/httprunner/httprunner/compare/v4.0.0...v4.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/httprunner/httprunner/releases/tag/v4.1.0'>v4.1.0</a>.</em>

**Comment by fengzhen2009** [comment]
![image](https://user-images.githubusercontent.com/18163842/171088684-486d0921-12c1-4e0b-86a3-0c8a748cacf6.png)
v4.1怎么没有startproject参数了的

**Reply by debugtalk** [reply]
在 v4 版本中，命令行工具需要使用 `hrp`
